python/test/unit/io/test_vtk.py

```diff
@@ -71,74 +74,80 @@ def test_save_3d_meshfunctions(tempfile, mesh_function_types, file_options,
     for d in range(mesh.topology.dim + 1):
         for t in mesh_function_types:
             mf = MeshFunction(t, mesh, mesh.topology.dim - d, type_conv[t](1))
-            VTKFile(tempfile + "mf.pvd", "ascii").write(mf)
-            f = VTKFile(tempfile + "mf.pvd", "ascii")
+            VTKFile(tempfile + "mf.pvd").write(mf)
+            f = VTKFile(tempfile + "mf.pvd")
             f.write(mf, 0.)
             f.write(mf, 1.)
-            for file_option in file_options:
-                VTKFile(tempfile + "mf.pvd", file_option).write(mf)
+            # for file_option in file_options:
+            #     VTKFile(tempfile + "mf.pvd", file_option).write(mf)
 
 
+@pytest.mark.xfail(reason="file_option not added to VTK initializer")
```

Comment:
But this turns off VTKFile testing. I am against xfailing whole test just because this file_options is not really supported.
Also, why is this done in this PR?

Reply:
Ok, I see VTK was not really tested anyway. But are these changes related to the PR?

Reply:
I can Remove the file format options from the tests, and thus activate them again. I Added these back in here because VTK is the only reliable format for higher order problems, and i figured that we should the support all output formats in VTK

Reply:
Let's register and issue to clean up the VTK interface from Python.